ElectronClient/app/gui/ConfigScreen.jsx

```diff
@@ -101,14 +101,27 @@ class ConfigScreenComponent extends React.Component {
 				const settings = Object.assign({}, this.state.settings);
 				settings[key] = event.target.value;
 				this.setState({ settings: settings });
-			}
+			};
 
 			return (
 				<div key={key} style={rowStyle}>
 					<div style={labelStyle}><label>{md.label()}</label></div>
 					<input type="text" style={controlStyle} value={this.state.settings[key]} onChange={(event) => {onTextChange(event)}} />
 				</div>
 			);
+		} else if (md.type === Setting.TYPE_INT) {
+			const onNumChange = (event) => {
+				const settings = Object.assign({}, this.state.settings);
+				settings[key] = event.target.value;
+				this.setState({ settings: settings});
+			};
+
+			return (
```

Comment:
Please make sure only tabs are used and indentation is correct.

Reply:
@winstc Can you please fix the issues with indentation and use tabs?